weldx/transformations.py

```diff
@@ -1004,6 +1061,13 @@ def as_euler(
         """
         return self.as_rotation().as_euler(seq=seq, degrees=degrees)
 
+    def reset_reference_time(self, time_ref_new):
+        if isinstance(time_ref_new, str):
+            time_ref_new = pd.Timestamp(time_ref_new)
```

Comment:
```suggestion
        if not isinstance(time_ref_new, pd.Timestamp):
            time_ref_new = pd.Timestamp(time_ref_new)
```
allow everything

Reply:
The issue here is, that with no extra instructions, the current exception tests for invalid inputs would fail. `None` as well as integers can be converted to `pandas.Timestamp`. Maybe we want to allow that, but we can discuss and change that another time.